auth-web/src/stores/business.ts

```diff
@@ -90,6 +90,8 @@ export const useBusinessStore = defineStore('business', () => {
       ...addConditionalProperty(resp.modified, { modified: resp.modified }),
       ...addConditionalProperty(resp.modifiedBy, { modifiedBy: resp.modifiedBy }),
       ...addConditionalProperty(resp.nrNumber, { nrNumber: resp.nrNumber }),
+      ...addConditionalProperty(resp.state, { status: resp.state }),
+      ...addConditionalProperty(resp.effectiveDate, { effectiveDate: resp.effectiveDate }),
```

Comment:
I tried to get the status and effectiveDate property. But the Business Object is still not getting the state and effectiveDate property by doing so, can I get some guidance or suggestions?

Reply:
take a look at the search api?
![image](https://github.com/user-attachments/assets/f029c896-8317-439d-b388-effc202732dc)

Reply:
> I tried to get the status and effectiveDate property. But the Business Object is still not getting the state and effectiveDate property by doing so, can I get some guidance or suggestions?

effectiveDate property is working! I assume the state/status is not updated in the api?

Reply:
The Auth API probably needs a small update to pass things along. I assume @vysakh-menon-aot will work on it next as part of his ticket.

Reply:
![image](https://github.com/user-attachments/assets/b465a481-8c42-4ca0-afa5-fa7f90e86b8f)